src/Model/AbstractManager.php

```diff
@@ -56,4 +56,11 @@ public function delete(int $id): void
         $statement->bindValue('id', $id, \PDO::PARAM_INT);
         $statement->execute();
     }
+    public function selectSearch(string $search): array
+    {
+        $query = "SELECT * FROM wine  WHERE name, year, price, 
+        description LIKE '% '. $search '.%'";
+
+        return $this->pdo->query($query)->fetchAll();
+    }
 }
```

Comment:
The SQL query in here, can't work. Is this function has really been tested ? Is it used anywhere ?